third_party/Elecard/MpegAudioEncLayer2/utils.cpp

```diff
@@ -482,16 +482,16 @@ void avcodec_string(char *buf, int buf_size, AVCodecContext *enc, int encode)
                  codec_name);
         switch (enc->channels) {
             case 1:
-                strcpy(channels_str, "mono"); // throws warning C4996 my be unsafe
+                strcpy_s(channels_str, strlen("mono"), "mono");
```

Comment:
All 4 of these should use sizeof(channels_str) for the second argument as it is supposed to be the size of the first argument. Also fix the indentation on the second one.

Reply:
Done